sigexport.py

```diff
@@ -410,6 +414,9 @@ def merge_with_old(dest, old):
 @click.option(
     "--source", "-s", type=click.Path(), help="Path to Signal source and database"
 )
+@click.option(
+    "--chat", "-c", type=click.Path(), help="Set of chat names to be exported"
```

Comment:
Delete the `type=` parameter, as it should be `STRING`, which is anyway the default.

Reply:
And maybe add what the chat 'names' are? Assume it's the contact name or group name?
e.g. `help="Comma-separated chat names to include. These are contact names or group names`.

Reply:
Sure, I have made those changes.